Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -753,7 +753,10 @@ public static void SetClientNotReady(NetworkConnection conn)
             {
                 if (LogFilter.Debug) Debug.Log("PlayerNotReady " + conn);
                 conn.isReady = false;
-                conn.RemoveObservers();
+                if (conn.playerController != null)
+                {
+                    conn.playerController.RemoveFromObservers();
```

Comment:
```suggestion
                    conn.playerController?.RemoveFromObservers();
```

Reply:
using this unity components gives warnings. apparently we are supposed to null check.